src/components/ProjectPreview.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTheme } from 'styled-components'
+import { PreviewText, PreviewWrapper, PreviewTitle } from './styles/ProjectPreview.styled'
+
+type Props = {
+  title: string
+  text: string
+  hasAnimation?: boolean
+  onClick?: () => void
+}
+
+const defaultProps = {
+  hasAnimation: false,
+  onClick: () => undefined,
+}
+
+export default function ProjectPreview(props: Props) {
+  const theme = useTheme()
+  const { h4, previewtitle } = { ...theme.typography.fontSize }
+  return (
+    <PreviewWrapper onClick={props.onClick}>
+      <PreviewTitle fontType={previewtitle}>{props.title}</PreviewTitle>
+      <PreviewText fontType={h4} hasAnimation={props.hasAnimation ?? false}>
```

Comment:
is the nullish coalescing operator necessary?

Reply:
acb0a0e9ab69f76346e7d80a15addbffef9a7514